src/apps/stellar/get_public_key.py

```diff
@@ -0,0 +1,41 @@
+from ubinascii import hexlify
+
+from trezor import ui
+from trezor.messages import ButtonRequestType
+from trezor.messages.StellarGetPublicKey import StellarGetPublicKey
+from trezor.messages.StellarPublicKey import StellarPublicKey
+from trezor.ui.text import Text
+
+from apps.common import seed
+from apps.common.confirm import confirm
+from apps.common.display_address import split_address
+from apps.stellar import helpers
+
+
+async def get_public_key(ctx, msg: StellarGetPublicKey):
+    node = await seed.derive_node(ctx, msg.address_n, helpers.STELLAR_CURVE)
+    pubkey = seed.remove_ed25519_prefix(node.public_key())
+
+    if msg.show_display:
```

Comment:
There is no device test for this case.

Reply:
Added in https://github.com/trezor/python-trezor/commit/e5cdf45956857468bc89cdf542d9e159cff9f2ad . I've just modified some of the existing tests to include show_display=True, I hope that's enough.

Reply:
Yes that's perfect, thanks!